homeassistant/components/icloud/account.py

```diff
@@ -161,6 +161,7 @@ def update_devices(self) -> None:
         """Update iCloud devices."""
         if self.api is None:
             return
+        _LOGGER.debug("Updating devices")
```

Comment:
If this is going into debug logging, could you please make the debug message a bit more expressive? Especially, if this will cause the "Someone logged in..." emails (which we all loathe).

Reply:
This message is related to updating devices.
If it kicks in a re-authentication (due to expired token), it will send that email, but i don't think this is the place to log that...
I think "Updating devices" is quite clear for when devices are being updated 😃 
This is mostly to make sure that polling is not happening (ie: this message does not show up in logs)

Reply:
But maybe it would be nice to have that message (logging in) to match the emails, though debug logging is disabled after some time, in integrations, so it probably needed INFO level.
I'll let one of the owners comment on that before using this PR for that